### PR TITLE
Spelling fixes for contrib section

### PR DESCRIPTION
**Pull request by kaedroho**

### CODE REVIEW

File: docs/contrib/staticsitegen.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ First, install ``django-medusa`` from pip:
 
 .. note::
 
-    Since 0.8.6, Wagtail uses StreamingHttpResponse to return Documents. `django-medusa`_ currently does not support this and the staticsitegen command will fail if your site contains Documents.
+    Since 0.8.6, Wagtail uses ``StreamingHttpResponse`` to return Documents. `django-medusa`_ currently does not support this and the ``staticsitegen`` command will fail if your site contains Documents.
     For a temporary fix, install this fork https://github.com/ctxis/django-medusa. (see also #1183)
```
Comment (tomdyson) [inline docs/contrib/staticsitegen.rst]: is this still correct?

File: wagtail/contrib/wagtailroutablepage/models.py [hunk 2]
```diff
@@ -37,8 +37,8 @@ def decorator(view_func):
 
 class RoutablePageMixin(object):
     """
-    This class can be mixed in to a Page subclass to allow urlconfs to be
-    embedded inside pages.
+    This class can be mixed in to a Page model to allow extra routes to be
```
Comment (tomdyson) [inline wagtail/contrib/wagtailroutablepage/models.py]: -> "a Page model, allowing extra routes to be added to it"

File: wagtail/contrib/wagtailroutablepage/models.py [hunk 3]
```diff
@@ -110,7 +109,7 @@ def resolve_subpage(self, path):
 
     def route(self, request, path_components):
         """
-        This hooks the subpage urls into Wagtails routing.
+        This hooks the subpage URLs into Wagtails routing.
```
Comment (tomdyson) [inline wagtail/contrib/wagtailroutablepage/models.py]: Wagtail's

File: wagtail/contrib/wagtailroutablepage/models.py [hunk 4]
```diff
@@ -135,8 +134,8 @@ def serve_preview(self, request, mode_name):
 
 class RoutablePage(RoutablePageMixin, Page):
     """
-    This class extends Page by adding methods to allow urlconfs
-    to be embedded inside pages
+    This class extends Page by adding methods to allow extra routes to be
```
Comment (tomdyson) [inline wagtail/contrib/wagtailroutablepage/models.py]: to allow -> which allow

File: docs/contrib/staticsitegen.rst (hunk 1 above)
Comment (kaedroho) [inline docs/contrib/staticsitegen.rst]: Nope, this has been removed on master

**Comment by tomdyson** [comment]
Happy for this to be merged with my changes.

**Pull request merged**